File: Nitesh/src/main/java/org/example/Employee.java
package org.example;

import java.util.Date;

public class Employee {
    private int id;
    private String name;
    private Address address;
    private int age;
    private Date joiningDate;
    private double salary;

    public Employee(int id, String name, Address address, int age, Date joiningDate, double salary) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.age = age;
        this.joiningDate = joiningDate;
        this.salary = salary;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public Address getAddress() {
        return address;
    }

    public int getAge() {
        return age;
    }

    public Date getJoiningDate() {
        return joiningDate;
    }

    public double getSalary() {
        return salary;
    }

    @Override
    public String toString() {
        return "Employee{id=" + id + ", name='" + name + "', address=" + address + ", age=" + age + ", joiningDate=" + joiningDate + ", salary=" + salary + "}";
    }
}

File: gayatri_patwe/oops-Assigement-1/Employee.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace oops_Assigement_1
{
    internal class Employee
    {

        public int Id { get; set; }
        public string Name { get; set; }
        public Address Address { get; set; }
        public int Age { get; set; }
        public DateTime JoiningDate { get; set; }
        public decimal Salary { get; set; }

        public Employee(int id, string name, Address address, int age, DateTime joiningDate, decimal salary)
        {
            Id = id;
            Name = name;
            Address = address;
            Age = age;
            JoiningDate = joiningDate;
            Salary = salary;
        }

        public override string ToString()
        {
            return $"ID: {Id}, Name: {Name}, Address: [{Address}], Age: {Age}, Joining Date: {JoiningDate.ToShortDateString()}, Salary: {Salary}";
        }

       

    }
}
